File: app/src/main/java/com/gigabytedevelopersinc/app/cometOTP/Receivers/BackupBroadcastReceiver.java
package com.gigabytedevelopersinc.app.cometOTP.Receivers;

import android.content.BroadcastReceiver;
import android.content.Context;

import com.gigabytedevelopersinc.app.cometOTP.Utilities.Settings;

public abstract class BackupBroadcastReceiver extends BroadcastReceiver {
    protected boolean canSaveBackup(Context context) {
        Settings settings = new Settings(context);
        return settings.isBackupLocationSet();
    }
}

File: app/src/main/java/com/gigabytedevelopersinc/app/cometOTP/Receivers/BackupBroadcastReceiver.kt
@file:Suppress("PackageName")
package com.gigabytedevelopersinc.app.cometOTP.Receivers

import android.content.BroadcastReceiver
import android.content.Context
import com.gigabytedevelopersinc.app.cometOTP.Utilities.Settings

abstract class BackupBroadcastReceiver : BroadcastReceiver() {
    protected fun canSaveBackup(context: Context): Boolean {
        val settings = Settings(context)
        return settings.isBackupLocationSet
    }
}
